content/api/hub/plugin_spin_cron_trigger.md

```diff
@@ -0,0 +1,66 @@
+title = "Spin Cron Trigger"
+template = "render_hub_content_body"
+date = "2022-10-15T00:22:56Z"
+content-type = "text/plain"
+tags = ["trigger", "rust"]
+
+[extra]
+author = "Fermyon"
+type = "hub_document"
+category = "Plugin"
+language = "Rust"
+created_at = "2023-12-05T00:22:56Z"
+last_updated = "2023-12-05T00:22:56Z"
+spin_version = “>=v2.0.1"
+summary = "An experimental cron trigger for Spin apps"
+url = "https://github.com/fermyon/spin-trigger-cron"
+keywords = "trigger, cron"
+
+---
+
+An experimental plugin that enables cron trigger for Spin applications. 
+
+### Prerequisites
+
+* [Rust](https://www.rust-lang.org/tools/install)
+* `pluginify` plugin (see below)
+
+```bash
+spin plugins install --url https://github.com/itowlson/spin-pluginify/releases/download/canary/pluginify.json
+```
+## Installing Template
+You can install the template using the following command:
+
+```bash
+spin templates install --git https://github.com/fermyon/spin-trigger-cron
+```
+
+Once the template is installed, you can create a new application using:
+
+```bash
+spin new -t cron-rust hello_cron --accept-defaults
+```
+
+To run the newly created app:
+
+```bash
+cd hello_cron
+spin build --up
+```
+
+### Modifying The Cron Trigger
```

Comment:
I wouldn't describe this as "modifying"... in fact... stand back... I'm gonna do it.  I'm gonna use THE C WORD 
```suggestion
### Configuring The Cron Trigger
```